metrics/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "metrics"
```

Suggestion to align with how (most) other crates are being named:

```suggestion
name = "datadog-metrics"
```